masonry/src/render_root.rs

```diff
@@ -188,8 +188,9 @@ impl RenderRoot {
                 // See https://github.com/linebender/druid/issues/85 for discussion.
                 let last = self.last_anim.take();
                 let elapsed_ns = last.map(|t| now.duration_since(t).as_nanos()).unwrap_or(0) as u64;
-
-                if self.root_state().request_anim {
+                let root_state = self.root_state();
+                if root_state.request_anim {
+                    root_state.request_anim = false;
```

Comment:
Is this a drive-by fix? Was this a bug previously?

Reply:
It was a bug previously, although it's not really a change in observable behaviour, just a missed optimisation. I don't think it's very important to land this more expediently than this change.

Reply:
(Was just a question, agree)